docker/wmagent/install.sh

```diff
@@ -51,8 +56,8 @@ mkdir -p $DEPLOY_DIR || true
 cd $BASE_DIR
 rm -rf deployment deployment.zip deployment-${DEPLOY_TAG};
 
-set -e 
-wget -nv -O deployment.zip --no-check-certificate https://github.com/dmwm/deployment/archive/$DEPLOY_TAG.zip
```

Comment:
Todor, if I pass `master` as the DEPLOY_TAG, this line seems to work just fine. And so it does for an old HG tag. Do we really need to change this github url?